node/src/chain_spec/content_config.rs

```diff
@@ -28,6 +28,7 @@ pub fn production_config() -> ContentConfig {
         platform_fee_percentage: Perbill::from_percent(1),
         auction_starts_at_max_delta: DAYS * 30,
         max_auction_whitelist_length: 100,
+        nft_limits_enabled: true,
```

Comment:
Just an observation: We can make use this feature instead of commenting out the checks in https://github.com/Joystream/joystream/pull/3914 once this PR is merged